src/modules/components/dao/daoDataListItem/daoDataListItem.tsx

```diff
@@ -0,0 +1,45 @@
+import type React from 'react';
+import { DataList, Heading, Icon, IconType } from '../../../../core';
+import { DaoAvatar, type IDaoAvatarProps } from '../daoAvatar';
+
+export interface IDaoDataListItemProps extends IDaoAvatarProps {
```

Comment:
Every data-list-item should extend the data-list-item-props interface (to be able, for instance, to pass a link to the data-list-item component). You can add the `name` and `logoSrc` properties for the avatar to the list of properties below
```suggestion
export interface IDaoDataListItemProps extends IDataListItemProps {
    name?: string;
    logoSrc?: string;
    [...]
```